package.json

```
@@ -6,7 +6,8 @@
   "main": "bl.js",
   "scripts": {
     "lint": "standard *.js test/*.js",
-    "test": "npm run lint && node test/test.js | faucet"
+    "test": "npm run lint && npm run test:typings && node test/test.js | faucet",
+    "test:typings": "npx tsc --allowJs --noEmit test/test.js"
```

```suggestion
    "test:types": "npx tsc --allowJs --noEmit test/test.js"
```
really minor nit, just feels more consistent across repos I personally work with (maybe there's a developing convention for the word "typings" that I've not seen though?)

---

It is okay to me to change it! I'm not going to accept suggestion (as Github suggests) just because I need to update also the `test` script to use `test:types` instead of `test:typings`, so I'll commit it soon 👍